v20.1/backup-and-restore.md

```diff
@@ -1,67 +1,96 @@
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BACKUP TO '<backup_location>';
+~~~
+
+If it's ever necessary, you can use the [`RESTORE`][restore] statement to restore your full cluster:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> RESTORE FROM '<backup_location>';
+~~~
 
-In most cases, it's recommended to use the [`BACKUP`][backup] command to take full nightly backups of your cluster:
+{{site.data.alerts.callout_info}}
+Note that a full cluster restore can only be run on a target cluster with no databases or tables.
+{{site.data.alerts.end}}
+
+Or to restore a  database:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<full_backup_location>';
+> RESTORE DATABASE bank FROM '<backup_location>';
 ~~~
 
-If it's ever necessary, you can then use the [`RESTORE`][restore] command to restore your cluster:
+Or to restore a table:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> RESTORE FROM '<full_backup_location>';
+> RESTORE TABLE bank.customers FROM '<backup_location>';
 ~~~
 
-### Manual full and incremental backups
+### Full and incremental backups
 
 If your cluster increases to a size where it is no longer feasible to take nightly full backups, you might want to consider taking periodic full backups (e.g., weekly) with nightly incremental backups. Incremental backups are storage efficient and faster than full backups for larger clusters.
 
 Periodically run the [`BACKUP`][backup] command to take a full backup of your database:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<full_backup_location>';
+> BACKUP TO '<backup_location>';
 ~~~
 
-Then create nightly incremental backups based off of the full backups you've already created.
+Then, create nightly incremental backups based off of the full backups you've already created:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<incremental_backup_location>'
-    INCREMENTAL FROM '<full_backup_location>', '<list_of_previous_incremental_backup_location>';
+> BACKUP TO '<backup_location>';
 ~~~
 
-If it's ever necessary, you can then use the [`RESTORE`][restore] command to restore your cluster:
+{{site.data.alerts.callout_info}}
+If you backup to a destination already containing a backup, an incremental backup will be produced in a subdirectory with a date-based name (e.g., `destination/day/time_1`, `destination/day/time_2`). For an example on how to explicitly specify the destination of an incremental backup, see [Backup and Restore - Advanced Options](backup-and-restore-advanced-options.html#incremental-backups-with-explicitly-specified-destinations)
+{{site.data.alerts.end}}
+
+If it's ever necessary, you can then use the [`RESTORE`][restore] command to restore your cluster, database(s), and/or table(s). [Restoring from incremental backups](restore.html#restore-from-incremental-backups) requires previous full and incremental backups. To restore from a destination containing the full backup, as well as the incremental backups (stored as subdirectories):
```

Comment:
`To restore from a destination containing the full backup, as well as the incremental backups (stored as subdirectories):`

I believe this only works if you took the backups with the "auto appending incremental" UX. If you use `BACKUP INCREMENTAL ... FROM ...` syntax and the incremental buckets happen to be subdirectories, I'm not sure it works. @dt do you know?

Reply:
Hmm, even if it does work technically, I don't think that we should encourage users to use the `INCREMENTAL FROM` syntax to create sub-directories manually. They need to be named a certain way, and this feels like a slippery slope that would let users shoot themselves in the foot too easily (e.g. it's much easier to fork or otherwise break the history chain, effectively breaking RESTORE).